src/clincoded/static/components/score/experimental_score.js

```diff
@@ -221,12 +234,27 @@ var ScoreExperimental = module.exports.ScoreExperimental = createReactClass({
         if (this.refs.scoreExplanation) {
             // Parse the score explanation entered by the curator
             let scoreExplanation = this.refs.scoreExplanation.getValue();
-            this.setState({scoreExplanation: scoreExplanation, formError: false}, () => {
+            this.setState({scoreExplanation: scoreExplanation, scoreError: false, scoreErrorMsg: ''}, () => {
                 this.updateUserScoreObj();
             });
         }
     },
 
+    // Check if changes has been made before saving the score object.
+    saveScore(e) {
+        if ((this.state.scoreStatus === this.state.origStatus ||
+             this.state.scoreStatus === 'none' && this.state.origStatus === null) &&
+            (this.state.modifiedScore === this.state.origScore ||
+             this.state.modifiedScore === 'none' && this.state.origScore === null) &&
+            (this.state.scoreExplanation === this.state.origScoreExplanation ||
+             this.state.scoreExplanation === '' && this.state.origScoreExplanation === null)) {
+                this.setState({scoreError: true, scoreErrorMsg: 'Cannot save because no field has been modified.  Please make your changes then save.'});
```

Comment:
What do you think about changing the `scoreErrorMsg` to 'Cannot save because the score/explanation has not been modified..'

Just a small suggestion for clarity!